### DISCUSSION TITLE
Nodata process is not working in multiple band image.

### DISCUSSION THREAD

**Discussion by geniuskim05** [developmentseed/titiler#338] [Q&A]
I have 7 band COG image(nodata value is zero).
There isn't nodata in 1 to 6 band, just only 7 band have nodata in land.
If I visualize 7 band only, not working like below image. The nodata option was set.
If I want to visualize just 7, how to solve it??
Is nodata coming out because other bands affect it?
Please answer me. Thank you.

![캡처](https://user-images.githubusercontent.com/65760090/125546783-895babae-ad58-493c-9f90-69716231733a.PNG)

**Comment by vincentsarago** [comment]
@geniuskim05 I'm not sure but in the nodata value is set per file not per band in the GeoTIFF spec. 

I'm not quite sure what you are doing, could you share your file and the requests you are doing ?
